src/neo/Network/P2P/Payloads/NotaryRequest.cs

```diff
@@ -0,0 +1,133 @@
+using Neo.IO;
+using Neo.Persistence;
+using Neo.SmartContract;
+using Neo.SmartContract.Native;
+using Neo.VM;
+using System;
+using System.IO;
+using System.Linq;
+
+namespace Neo.Network.P2P.Payloads
+{
+    public class NotaryRequest : IInventory
+    {
+        /// <summary>
+        /// Represents the fixed value of the <see cref="Transaction.Script"/> field of the fallback transaction.
+        /// </summary>
+        public static readonly byte[] FallbackFixedScript = new byte[] { (byte)OpCode.RET };
```

Comment:
Please, do not use this fixed value to validate fallback transactions. Fallback transaction is allowed to have *any* valid script inside it, the user of the Notary subsystem should have the ability to make use of the fallback in case if something went wrong and the main transaction wasn't completed.

Reply:
OK.I will delete it.